README.md

```diff
@@ -732,7 +732,8 @@ IEEE Winter Conf. Applications of Computer Vision (WACV), 2019.
 - <a name="Chen19fnins"></a>Chen, G., Chen, J., Lienen, M., Conradt, J., Roehrbein, F., Knoll, A.C.,  
 *[FLGR: Fixed Length Gists Representation Learning for RNN-HMM Hybrid-Based Neuromorphic Continuous Gesture Recognition](https://dx.doi.org/10.3389%2Ffnins.2019.00073)*,  
 Front. Neurosci., 2019, 13:73.
-
+- <a name="DBLP:journals/corr/abs-1803-10106"></a>Gregor Lenz, SioHoi Ieng, Ryad Benosman
+*[Event-based Dynamic Face Detection and Tracking Based on Activity](http://arxiv.org/abs/1803.10106)*, arXiv, 2019. [YouTube](https://www.youtube.com/watch?v=F5UzXQsr5Es)
```

Comment:
Need to format the entry as other ones:
- author names are formatted as: LastName, Initial., 
- leave to spaces at the end of the line for github to put a line break
- the anchor name should be name="Lenz19arxiv"